tutorials/nservicebus-101/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "NServiceBus 101: Messaging Basics"
+suppressRelated: true
+---
+
+In this course, you will learn how to get working quickly with NServiceBus.
+
+
+## Objectives
+
+In this course we'll build a messaging system using a retail e-commerce domain.
```

Comment:
How about "using a retail e-commerce domain" -> "for a retail e-commerce domain" ?